framework/src/transfers/MultiAppGeneralFieldTransfer.C

```diff
@@ -1215,6 +1215,24 @@ MultiAppGeneralFieldTransfer::getToVarName(unsigned int var_index)
   return var_name;
 }
 
+Point
+MultiAppGeneralFieldTransfer::getMaxToProblemsBBoxDimensions() const
+{
+  Point max_dimension = {std::numeric_limits<Real>::min(),
+                         std::numeric_limits<Real>::min(),
+                         std::numeric_limits<Real>::min()};
+
+  for (auto & to_mesh : _to_meshes)
+  {
+    const auto bbox = to_mesh->getInflatedProcessorBoundingBox();
+    for (auto dim : make_range(LIBMESH_DIM))
+      max_dimension(dim) = std::max(
+          max_dimension(dim), std::max(std::abs(bbox.first(dim)), std::abs(bbox.second(dim))));
```

Comment:
Not at all what I'd have expected; I'd definitely have used BBox dimensions, not max(bbox-coordinate-distance-from-0) ... but I can't actually think of any reason why this won't work just as well, except in perverse cases where we'd expect Point roundoff error to be exceeding other codepath tolerances for other reasons too.

Reply:
mesh could be far from the origin yet have a tiny extent?
what if the extent of the mesh in a dimension ends up being the fruit of numerical errors (like a single point is not well rounded in Z, making a mesh of size 1e-17 or so)

Reply:
Far from the origin but with a tiny extent would break in other ways first, I think.

OH!!!  We ought to be using a single scale here, not a per-dimension scale, don't we?  If we're working on a 2D manifold that's on a cartesian plane then we're going to get `numeric_limits<Real>::min()` for the normalization in the perpendicular dimension, but if we're exchanging data with a 3D mesh that might actually be perturbing that point in the perpendicular dimension then we need a lot more tolerance than that.

Reply:
this remains fully on the target side though.
It doesnt matter what the origin mesh looks like, this map is for comparing:
- point in target mesh
- point + app_position - app_position in target mesh

Reply:
Okay, that seems to make sense.  And I can't seem to find any other failure cases.